### PR TITLE
formatting, adding dependencies, resolving eslint warnings

### PR DESCRIPTION
**Pull request by paolovic**

### CODE REVIEW

File: src/components/views/PublicLobbies.js [hunk 1]
```diff
@@ -53,20 +53,20 @@ const PublicLobbies = () => {
...
-                if (error.response.status == 404) {
-                    setErrorMessage("This lobby does not seem to be live!");
-                } 
-                
-                else if (error.response.status == 409) {
-                   setErrorMessage("This lobby is already full!");
-                }
-
-                else {
+            if (error.response.status === 404) {
```
Comment (paolovic) [inline src/components/views/PublicLobbies.js:59]: here are the "important changes", rest is formatting

File: src/components/views/PublicLobbies.js [hunk 2]
```diff
@@ -53,20 +53,20 @@ const PublicLobbies = () => {
...
+                setErrorMessage("This lobby does not seem to be live!");
+            }
+
+            else if (error.response.status === 409) {
```
Comment (paolovic) [inline src/components/views/PublicLobbies.js:63]: here are the "important changes", rest is formatting

File: src/components/views/game/Game.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
 import React, {useEffect, useState} from "react";
 import {Helmet} from "react-helmet";
-import Map from "../../fragments/game/Map";
```
Comment (paolovic) [inline src/components/views/game/Game.js:3]: here are the "important changes", rest is formatting

File: src/components/views/lobby/Lobby.js [hunk 4]
```diff
@@ -189,75 +191,75 @@ const setClassName = (user) => {
         }
 
         fetchData();
-    }, []);
+    }, [id, token]);
```
Comment (paolovic) [inline src/components/views/lobby/Lobby.js:194]: here are the "important changes", rest is formatting

File: src/components/views/lobby/ShareQRCode.js [hunk 5]
```diff
@@ -35,7 +35,7 @@ const ShareQRCode = ({ id }) => {
         }
 
         fetchData();
-    }, []);
+    }, [id, token]);
```
Comment (paolovic) [inline src/components/views/lobby/ShareQRCode.js:38]: here are the "important changes", rest is formatting

File: src/components/views/lobby/UpdateLobby.js [hunk 6]
```diff
@@ -59,7 +59,7 @@ const UpdateLobby = ({id}) => {
         }
 
         fetchData();
-    }, []);
+    }, [id, token]);
```
Comment (paolovic) [inline src/components/views/lobby/UpdateLobby.js:62]: here are the "important changes", rest is formatting

**Review by luis-tm — APPROVED** [review]
much better now! ty

File: src/components/views/lobby/Lobby.js (hunk 4 above)
Comment (luis-tm) [reply src/components/views/lobby/Lobby.js:194]: ah cool! I saw the warnings related to this, but I didn't know how to fix them! :)

**Review by apple00juice — APPROVED** [review]
Looks good 👍

**Pull request merged**